.github/workflows/lib-injection.yml

```diff
@@ -106,4 +106,4 @@ jobs:
       - name: Output app logs (LOOK HERE IF THE JOB FAILS)
         if: success() || failure()
         run: |
-          docker-compose logs
+          docker-compose logs || true
```

Comment:
I saw that the issue is from `docker-compose: command not found` from the link you shared. Is `docker-compose` not guaranteed to be installed every time?